releasenotes/config.yaml

```diff
@@ -1,3 +1,49 @@
 ---
 encoding: utf8
 default_branch: main
+collapse_pre_releases: true
+sections:
+    - [features, New Features]
+    - [features_algorithms, Algorithms Features, 2]
+    - [features_circuits, Circuits Features, 2]
+    - [features_primitives, Primitives Features, 2]
+    - [features_providers, Providers Features, 2]
+    - [features_pulse, Pulse Features, 2]
+    - [features_qasm, OpenQASM Upgrade Notes, 2]
+    - [features_qpy, QPY Features, 2]
+    - [features_quantum_info, Quantum Information Upgrade Notes, 2]
```

Comment:
Aside from the "can't reliably embed section headings in reno" thing we talked about offline, the other issue is that there's two section headings called "Quantum Information Upgrade Notes", because these two categories up here have typos in them.

Reply:
aha!!! thanks for catching this, I really couldn't find those typos and was wondering where the warning was coming from.